docs/configuration.md

```diff
@@ -23,6 +23,7 @@ Some less common environment variables are:
 | --------------------- | --------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `GHE_HOST`            | `new Probot({ Octokit: ProbotOctokit.defaults({ baseUrl }) })`        | The hostname of your GitHub Enterprise instance. <p>_(Example: `github.mycompany.com`)_</p>                                                                                                                                                                                                                 |
 | `GHE_PROTOCOL`        | `new Probot({ Octokit: ProbotOctokit.defaults({ baseUrl }) })`        | The protocol of your GitHub Enterprise instance. Defaults to HTTPS. Do not change unless you are certain. <p>_(Example: `https`)_</p>                                                                                                                                                                       |
+| `GHE_ORG`        | -                                                                          | The organization where you want to register the app in the app creation manifest flow. If set, the app is registered for an organization (https://github.com/organizations/ORGANIZATION/settings/apps/new), if not set, the GitHub app would be registered for the user account (https://github.com/settings/apps/new).                                                                                                                                                                      |
```

Comment:
I'm not sure about the name, I think GHE stands for GitHub enterprise edition, but I'm not sure. @gr2m what does GHE actually mean?

Reply:
just GitHub Enterprise

Reply:
Then do you think it should be GH_ORG or GHE_ORG?

Reply:
yeah, good point.

I'm not sure this PR is the right approach in the first place, I'll need a moment to think about it. I comment on it soon

Reply:
Let's do `GH_ORG`. Sorry for the delayed response

Reply:
Thank you! I have made the change.